cmd/get.go

```diff
@@ -558,21 +548,17 @@ func init() {
 		"project name; if not specified will be inferred from git origin")
 	GetPplCmd.Flags().StringP("project-id", "i", "",
 		"project id; if not specified will be inferred from git origin")
-	GetPplCmd.Flags().Int64P("created-after", "", 0,
-		"filter for pipeline creation timestamp; by default, this is (now - 90d)")
-	GetPplCmd.Flags().Int64P("created-before", "", 0,
-		"filter for pipeline creation timestamp; by default, this is now")
+	GetPplCmd.Flags().DurationP("age", "", DefaultListingAge,
+		"filter for listing pipelines based on age; by default, we list only pipelines created in the last 90 days")
 	getCmd.AddCommand(GetPplCmd)
 
 	getCmd.AddCommand(GetWfCmd)
 	GetWfCmd.Flags().StringP("project-name", "p", "",
 		"project name; if not specified will be inferred from git origin")
 	GetWfCmd.Flags().StringP("project-id", "i", "",
 		"project id; if not specified will be inferred from git origin")
-	GetWfCmd.Flags().Int64P("created-after", "", 0,
-		"filter for pipeline creation timestamp; by default, this is (now - 90d)")
-	GetWfCmd.Flags().Int64P("created-before", "", 0,
-		"filter for pipeline creation timestamp; by default, this is now")
+	GetWfCmd.Flags().DurationP("age", "", DefaultListingAge,
+		"filter for listing pipelines based on age; by default, we list only pipelines created in the last 90 days")
```

Comment:
Maybe we should add here an example so it is clear that you should specify the value in hours.